### PR TITLE
[TASK] My Orders: Change sorting arrows (RT-2950)

### PR DESCRIPTION
**Pull request by darkdarkdragon**
My Orders: Change sorting arrows to match size
and shape of other arrows on overview page

### CODE REVIEW

**Comment by darkdarkdragon** [comment]
rt-2950-after2

**Comment by rippletravis** [comment]
Looks good visually.

**Comment by annatonger** [comment]
@mrajvanshy @MatthewPhinney @yongsoo

**Comment by mrajvanshy** [comment]
This still needs to be rebased

File: src/jade/tabs/trade.jade [hunk 1]
```diff
@@ -449,12 +453,12 @@ section.col-xs-12.content(ng-controller="TradeCtrl")
                   .col-md-2(l10n) Action
               .row(ng-repeat="entry in offers | rpcurrentpair:sort_options.current_pair_only:order.prev_settings | rpvalues | rpsortmyorders:sort_options.sort_field:sort_options.reverse track by entry.seq", ng-hide="entry.isZero",
                 ng-class="{cancelling: entry.cancelling || (editOrder.cancelling && entry.seq == editOrder.seq)}")
-                .col-md-1.type(data-label="Type")
+                .col-md-1.type(l10n-data-label="Type")
```
Comment (jublo) [inline src/jade/tabs/trade.jade]: Thanks for making these localizable. Good job!

File: src/jade/tabs/widgets/orders.jade [hunk 2]
```diff
@@ -30,26 +30,30 @@
       .my
         div
           .row.head.hidden-xs.hidden-sm
-            .col-md-1.type(style="cursor: pointer;", l10n, rp-orders-sort-header="sort_options.sort_field",
```
Comment (jublo) [inline src/jade/tabs/widgets/orders.jade]: Can/should these inline styles be extracted to LESS?

Comment (darkdarkdragon) [inline src/jade/tabs/widgets/orders.jade]: @jublo  They are. You are looking at removed line :)

**Comment by vhpoet** [comment]
rebase please

**Comment by darkdarkdragon** [comment]
@vhpoet done

**Pull request merged**